libs/data-types/src/lib/decorators/FixedValue.ts

```diff
@@ -0,0 +1,12 @@
+import { CoscradDataType } from '../types';
+import appendMetadata from '../utilities/appendMetadata';
+import mixinDefaultTypeDecoratorOptions from './common/mixinDefaultTypeDecoratorOptions';
+import { TypeDecoratorOptions } from './types/TypeDecoratorOptions';
+
+export function FixedValue(userOptions: TypeDecoratorOptions): PropertyDecorator {
```

Comment:
We either need to pass the value into the decorator factory, or find it via reflection.

Reply:
Either way, it needs to be part of the metadata and to flow through as a validation parameter.